src/core/components/people/index.tsx

```diff
@@ -0,0 +1,107 @@
+import PersonIcon from '@mui/icons-material/Person';
+import { Avatar, Box, Divider, Grid, List, ListItem, ListItemAvatar, ListItemText, Typography } from "@mui/material";
+import { styled } from '@mui/material/styles';
+import { FunctionComponent } from 'react';
+
+const HorizontalCenterContainer = styled(Box)(({
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "center",
+}));
+
+
+const PeopleClass: FunctionComponent = () => {
+    const data = [
+        {
+            name: "1",
+            role: "teacher"
+        },
+        {
+            name: "2",
+            role: "teacher"
+        },
+        {
+            name: "3",
+            role: "student"
+        },
+        {
+            name: "4",
+            role: "student"
+        },
+        {
+            name: "5",
+            role: "student"
+        },
+        {
+            name: "6",
+            role: "student"
+        },
+        {
+            name: "7",
+            role: "student"
+        },
+    ]
+    const teachers =data.filter((e)=>e.role==="teacher")
+    const students=data.filter((e)=>e.role==="student")
+    return (
+        <>
+            <HorizontalCenterContainer>
+                <Grid item xs={7} sx={{ width: "100%" }}>
```

Comment:
Go to 'src/shared/constants.ts' and define new constant like `export const PEOPLE_GRID_WIDTH=7`. Then import and change to `<Grid item xs={PEOPLE_GRID_WIDTH} sx={{ width: "100%" }}>`